_includes/project_button.html

```
@@ -1,11 +1,4 @@
-<!-- <!DOCTYPE html> -->
-<html>
-  <body>
-    <div class="center" style="margin-bottom: 30px;">
-      <script>
-        var href_link = ('../' + "{{ page.title | replace: " ", "_" }}" + '/').toLowerCase();
-        document.write('<a class="btn-group" href="' + href_link + '">{{ page.title }}</a>');
-      </script>
-    </div>
-  </body>
-</html>
+{% assign href_link = page.title | replace: ' ', '_' | downcase | prepend: '../' | append: '/' %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)